coverage.py

```diff
@@ -333,7 +335,9 @@ def process_item(self, identifier):
             # CoverageProvider didn't try to create a Work, or that a
             # preexisting Work has been removed. In the name of
             # resiliency, we might as well try creating a Work.
-            work, is_new = license_pool.calculate_work(even_if_no_author=True)
+            work, is_new = license_pool.calculate_work(
+                even_if_no_author=True, even_if_no_title=True
+            )
```

Comment:
This is the main improvement that actually helped things. Now, looking up an Identifier that's had all its work done, but doesn't have a Work, will create the Work. However, we still need to make sure all those Identifiers get Works ahead of time, because creating a Work can be very expensive in certain cases.